src/Microsoft.Android.Sdk.ILLink/SubStepDispatcher.cs

```diff
@@ -1,10 +1,14 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using Mono.Linker.Steps;
 
 namespace Microsoft.Android.Sdk.ILLink
 {
-	public class SubStepDispatcher : Mono.Linker.Steps.SubStepsDispatcher
+	public class SubStepDispatcher : MarkSubStepsDispatcher
```

Comment:
Why is this needed?

Reply:
Because the dispatcher only accepts a list of steps in its protected constructor.